dev/run

```diff
@@ -248,6 +253,8 @@ def hashify(pwd, salt=COMMON_SALT, iterations=10, keylen=20):
 def startup(ctx):
     atexit.register(kill_processes, ctx)
     boot_nodes(ctx)
+    if ctx['no_join']:
+        return
```

Comment:
I think need to make this check after `ensure_all_nodes_alive` - this step makes sure that nodes booted on previous step are ready for requests.